nlp_apps.ipynb

```diff
@@ -797,11 +797,215 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": true
    },
+   "source": [
+    "## Text Analysis using TF-IDF\n",
+    "Since we know that computers are great with numbers but they cannot work out with the natural language. So in-order to overcome that text can be directly converted to numbers for analyzing.One of the most common and popular technique for this is TF-IDF which stands for Term Frequency and Inverse Document Frequency. \n",
+    "\n",
+    "1. **TF(Term -Frequency):-**Gives the frequency of each word in a document. As the number of occurances of a word increases in a document its value increases for that document. Basically, if a word appears more times in a docuemnt then that word is important to that document. \n",
+    "    \n",
+    "    **TF(t) = (Number of times term t appears in a document) / (Total number of terms in the document)**\n",
+    "\n",
+    "\n",
+    "2. **IDF(Inverse Document Frequency):-** It calculates, how much a word is important for a given document. The words that occur in less documents are more important as compared to the words that occur in more number of documents. It helps to find the important words across the documents .\n",
```

Comment:
The first comma is not needed. Then, it is 'fewer documents' instead of 'less documents', the 'as' part in 'as compared' is a bit awkward and can be removed, and you should say 'in more documents' and not 'in more number of documents'. Finally, you added a space before the final period.